### ISSUE TITLE
Accessibility : Screen reader is not reading the "searchForMoreText" prop value in the Tag Picker Suggestion pop up

### ISSUE THREAD

**Issue by v-sagsah** [microsoft/fluentui#13291]
**Browser Details: ​**
OS: Windows 10 (Version 1909, Build: 18363.778)
Microsoft Edge Dev Version 84.0.502.0 (Official build) dev (64-bit)
Chrome Version 81.0.4044.138 (Official Build) (64-bit)

**Issue:**
Screen reader is not reading the "searchForMoreText" prop value in the Tag Picker Suggestion pop up.

_**Note :** Issue can be reproducible in both Narrator/NVDA._

Please find the below code pen link to repro the issue:
https://codepen.io/sagsah/pen/RwWxxpw?editors=1010
https://codepen.io/slimemold/pen/XWmOVpZ?editors=0010

Please find the below NVDA speech logs. 
**Issue:** As you can see, when focus goes to "Load all results", screen reader is reading different message.
**Expected:** Screen reader should be reading "Load all results".

![Screenshot (111)](https://user-images.githubusercontent.com/64409491/82680004-4f0d6b00-9c69-11ea-8246-4bdbe7d2943a.png)

**Comment by aneeshack4** [comment]
test

**Comment by ecraig12345** [comment]
Looked into this a little more and I think I see what you mean here: if you **down arrow through the options** down to the bottom, the button at the bottom (where `searchForMoreText` goes) is visually selected and can be activated with enter, but it's never read. That definitely seems wrong.

**Comment by v-sagsah** [comment]
@ecraig12345 con you please update on when this bug will be fixed ?

**Comment by ecraig12345** [comment]
I did look into this a bit but the fix was not obvious and may be non-trivial, which is why I marked this as needing backlog review instead of fixing it now. I can't give a timeline at this point because we're currently focused on larger work items around performance and coherence for version 8.

**Comment by v-sagsah** [comment]
@ecraig12345 / @aneeshack4 Is there any update on these particular issue ?
As we have a release on July 13th, & we have go through 1CS assessments, we need to know the ETA for this issue, so that we can plan our release accordingly. Without fixing these issue our accessibility grade can be impacted, so expecting a quick resolution on the same.

**Comment by v-sagsah** [comment]
@smhigley Thank you for the fix. We could see your PR is still in open status. Is there any way to expedite the process ?  As we are approaching to our release date & we have to go through 1CS assessments. Without fixing these issue our accessibility grade can be impacted.

**Comment by v-sagsah** [comment]
@ecraig12345 / @smhigley Request you to please update on these ? Is it possible to make the fix available on or before 29th June ?

**Comment by paulgildea** [comment]
@ecraig12345 @smhigley - Any status on the PR?

**Comment by paulgildea** [comment]
@ecraig12345 @smhigley - Any updates on this PR?
